### PR TITLE
make bot ignore non-bot jobs

### PR DESCRIPTION
**Pull request by Hafsa-Naeem**

### CODE REVIEW

**Comment by boegel** [comment]
@Hafsa-Naeem We should move the code block that creates the symlink and the code block that releases the job *after* checking whether the metadata file exists in `process_new_job`

File: eessi_bot_job_manager.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import re
 import subprocess
 import time
+import os.path
```
Comment (trz42) [inline eessi_bot_job_manager.py:30]: Just curious, do we need to import `os.path`? `os` is imported already (a few lines above).

Comment (boegel) [reply eessi_bot_job_manager.py:30]: Then no, just `import os` is sufficient to use `os.path`

**Comment by boegel** [comment]
closing in favor of #63

**Pull request closed without merging**